source/stylesheets/all_styles.scss

```diff
@@ -1,4 +1,54 @@
 @import 'mediaqueries/media-header';
+@import 'mediaqueries/media-main-content';
 @import 'variables';
 @import 'layout/header';
+@import 'layout/main-content/slider';
+@import 'layout/form';
 @import 'components/button';
+@import 'mediaqueries/media-form';
+
+// import fontello icons
+@import 'icons/fontello';
+@import 'icons/fontello-ie7';
+@import 'icons/fontello-ie7-codes';
+@import 'icons/fontello-codes';
+@import 'icons/fontello-embedded';
+
+
+.sections {
+  text-align: center;
+
+  button {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .blue-button,
+  .dark-blue-button {
+    font-size: $small-font;
+    padding: 12px 35px;
+    margin: 10px auto;
+  }
+}
+
+.section-content {
+  margin: 50px 0;
+
+}
+
+.section-heading {
+  text-transform: uppercase;
+  font-size: $medium-font;
+}
+
+.section-paragraph {
+  padding-bottom: 10px;
+  max-width: 90%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.section-icon {
+  height: 130px;
```

Comment:
actually it's not a real icon within "i" or "span". That's png image, I think that would be better to rename it